.gitlab-ci.yml

```diff
@@ -142,7 +142,7 @@ before_script:
       -e local_release_dir=${PWD}/downloads 
       -e etcd_deployment_type=${ETCD_DEPLOYMENT} 
       -e kubelet_deployment_type=${KUBELET_DEPLOYMENT} 
-      cluster.yml;
+      cluster.yml --skip-tags=download --limit=k8s-cluster:kube-master;
```

Comment:
actually that was a bad idea to limit the idempotency checks. let's just make them to be running shorter than 60 min in total, somehow...